llvm/unittests/BinaryFormat/ELFTest.cpp

```diff
@@ -0,0 +1,32 @@
+//===- ELFTest.cpp ----------------------------------------------*- C++ -*-===//
```

Comment:
Nit: I believe the `C++` marker only needs to be in headers.

Reply:
`-*- C++ -*-` is indeed a rare exception, so I am removing the marker.
```
% rg 'C\+\+ \*-=' -l -g '*.cpp'
llvm/tools/llvm-pdbutil/PrettyFunctionDumper.cpp
llvm/tools/llvm-pdbutil/PrettyBuiltinDumper.cpp
llvm/tools/llvm-pdbutil/PrettyTypeDumper.cpp
llvm/tools/llvm-pdbutil/PrettyExternalSymbolDumper.cpp
llvm/tools/llvm-pdbutil/PrettyCompilandDumper.cpp
llvm/tools/llvm-pdbutil/PrettyTypedefDumper.cpp
llvm/lib/CodeGen/GlobalISel/GISelKnownBits.cpp
llvm/lib/MC/SPIRVObjectWriter.cpp
```